test/runtests.jl

```diff
@@ -3,7 +3,9 @@ using LinearAlgebra
 using SparseArrays
 using Statistics: mean
 
-using StaticArrays: MVector
+using StaticArrays: MVector, @SVector, SA
+
+using Unitful
```

Comment:
```suggestion
using Unitful: @u_str
```

To clarify what we use in the tests